src/App.tsx

```diff
@@ -1,26 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handleAmountOfItems = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ): void => {
+    const { value } = event.target;
+
+    setCurrentPage(1);
+    setPerPage(+value);
+  };
+
+  const handlePage = (event: React.MouseEvent<HTMLAnchorElement>): void => {
+    const { dataset } = event.currentTarget;
+
+    setCurrentPage(Number(dataset.value));
+  };
+
+  const startPos = perPage * currentPage - perPage + 1;
+
+  const endPos = perPage * currentPage > 42
```

Comment:
Consider using a constant for the magic number '42' to give it a meaningful name.